### ISSUE TITLE
gui: screen lockout - keys ignored / Escape does nothing - not clear how to go back to home

### ISSUE THREAD

**Issue by sa2000a** [plexinc/plex-media-player#731]
### Test environment
**PMS Version:** 1.13.0.5023-31d3c0c65
**PMP Version:** 2.10.0.849-e02dbeca build date: 2018-05-01 
**Platform (Windows/OSX/Embedded):** windows 10

### Steps to reproduce
1. Not sure - I was trying to reproduce an issue with direct play playback - stopping / start with resume repeatedly and at some point ended up with the screen below

### Current behavior
1. All keys ignored. All mouse actions ignored. Not obvious how to recover. Log shows keys ignored

### Expected behavior
1. Escape should always work. Mouse click should bring options to exit / return

![image](https://user-images.githubusercontent.com/6320013/39960264-7272b8ce-5617-11e8-943d-c41fca562820.png)

switching mode worked but still nothing
![image](https://user-images.githubusercontent.com/6320013/39960270-84f135e8-5617-11e8-8b98-fd5d26a3cc2e.png)

Logs
[logs.zip](https://github.com/plexinc/plex-media-player/files/1997817/logs.zip)

conf files
[Conf files.zip](https://github.com/plexinc/plex-media-player/files/1997819/Conf.files.zip)

**Comment by markus101** [comment]
Which screen were you playing from?
Have you repeated this issue multiple times?

Recovering from that state is going to depend what state the app is in, without being able to reproduce it it's going to be impossible to know for sure what happened to cause it. Any additional information you have would be helpful.

**Comment by dan-the-man-nz** [comment]
longchair says my issue 732 is the duplicate of this

i had the same issue i described in my issue, last night again, pressing back from the "playing next" screen

**Comment by markus101** [comment]
Not sure that's the same issue, though similar end result. This issue looks like like the app crashed and yours looks like focus was targeted on something that no longer existed.

**Comment by sa2000a** [comment]
I did not think mine was a crash. The logs show the keyboard input being processed and ignored
It sounded just like #732 

What i was doing is trying to reproduce a direct play issue with resumes that a user reported
So i was doing a lot of play, pause, stop, play / resume, I kept on doing this trying to see if i get the user issue. I did not. At some point it did go to next episode. and I may have gone back. Can;t remember exact steps - but have attached all logs
